android/KMEA/app/src/main/java/com/tavultesoft/kmea/packages/LexicalModelPackageProcessor.java

```diff
@@ -27,7 +22,11 @@
 public class LexicalModelPackageProcessor extends PackageProcessor {
   private static final String TAG = "LMPackageProcessor";
 
-  static File constructPath(File path, boolean temp) {
+  public LexicalModelPackageProcessor(File resourceRoot) {
+    super(resourceRoot);
+  }
+
+  public File constructPath(File path, boolean temp) {
```

Comment:
I feel like this one could be based near-directly on its base class's code; the only real difference is use of `KMManager.KMDefault_LexicalModelPackages` over `KMManager.KMDefault_AssetPackages`.  Nothing a 'default parameter' or a `protected` helper version couldn't help fix.

Reply:
Got it. I'll add an optional parameter for `KMManager.KMDefault_AssetPackages`.

Reply:
Ah, so it'd take a bit more work then.  It can always be set as a constructor argument, but... sure, I won't let that be a blocker, then.